beard/clustering/blocking.py

```diff
@@ -99,52 +158,38 @@ def _validate(self, X, blocks):
 
     def _fit(self, X, y, blocks):
         """Fit base clustering estimators on X."""
+        def _grouper(jobs, seq):
```

Comment:
Sure, the only thing that I don't like is this underscore. Usually it means that the authors don't want it to be imported.